pennylane/measurements/mid_measure.py

```diff
@@ -100,11 +100,11 @@ def circuit(x, y):
                 qml.expval(m0), qml.var(m0), qml.probs(op=m0), qml.counts(op=m0), qml.sample(m0)
             )
 
-    >>> circuit(1.0, 2.0, shots=10000)
+    >>> circuit(1.0, 2.0, shots=1000)
     (0.702, 0.20919600000000002, array([0.298, 0.702]), {0: 298, 1: 702}, array([0, 1, 1, ..., 1, 1, 1]))
```

Comment:
This number of shots can be inferred from the `counts` result: `298+702=1000` instead of 10k.